doge-webapp/package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "doge-webapp",
+    "version": "0.0.1",
+    "description": "doge-webapp",
+    "scripts": {
+        "test": "NODE_PATH='src/main/resources/static/js' node_modules/.bin/mocha 'src/test/js/**/*.js'"
```

Comment:
This is the "magic."
1.  Add our webapp's JS directory as the `NODE_PATH`
2.  Run `mocha`
3.  Tell it to run the tests under `src/test/js`